zerver/openapi/zulip.yaml

```diff
@@ -9514,11 +9514,18 @@ paths:
   /users:
     get:
       operationId: get-users
-      summary: Get all users
+      summary: Get users
       tags: ["users"]
       description: |
-        Retrieve details on all users in the organization. Optionally
-        includes values of [custom profile fields](/help/custom-profile-fields).
+        Retrieve details on users in the organization.
+
+        The `user_ids` query parameter can be used to limit the results to
+        specific users.
+
+        Optionally includes values of [custom profile fields](/help/custom-profile-fields).
+
+        **Changes**: New in Zulip 11.0 (feature level ZF-a0b2d9). Clients can now
+        specify a list of `user_ids` to fetch.
 
         You can also [fetch details on a single user](/api/get-user).
```

Comment:
This shouldn't be in the "Changes" section, because it's relevant even for a reader that's interested only in the latest version of the API.